jerry-debugger/jerry-client-ws.py

```diff
@@ -678,6 +707,17 @@ def parse_source(debugger, data):
         for line, breakpoint in function.lines.items():
             debugger.line_list.insert(line, breakpoint)
 
+    # Try to set the pending breakpoints
+    if len(debugger.pending_breakpoint_list) != 0:
+        logging.debug("Pending breakpoints list: %s" % (debugger.pending_breakpoint_list))
+
+        for breakpoint in debugger.pending_breakpoint_list:
+            if type(breakpoint) == int:
+                breakpoint = source_code_name + ":" + str(breakpoint)
+            set_breakpoint(debugger, False, breakpoint)
```

Comment:
The set_breakpoint should not complain if the breakpoint is still not found (silent mode).